### PR TITLE
Added team and game_teams tests and initializations

### PR DESCRIPTION
**Pull request by freeheeling**
@sasloan @zacisaacson @mcat56 

Let me know what you think.

### CODE REVIEW

File: runner.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require './lib/stat_tracker'
 
-game_path = './data/games.csv'
+game_path = './data/games_dummy.csv'
```
Comment (mcat56) [inline runner.rb:3]: Not sure what this dummy file is?

**Review by ghost — APPROVED** [review]
This looks good to me bro!! lets load it!!

**Review by mcat56 — COMMENTED** [review]
Pending that dummy file change everything looks good!

File: runner.rb (hunk 1 above)
Comment (freeheeling) [reply runner.rb:3]: Sorry about that. I thought I had deleted that dummy file. I'll look into it. Thanks.

Comment (ghost) [reply runner.rb:3]: good catch Mary!!

Comment (mcat56) [reply runner.rb:3]: Okay, sounds great! Thank you @sasloan !!

**Pull request closed without merging**